src/httpClient.ts

```diff
@@ -41,79 +52,130 @@ export default class httpClient {
-                sampling_start_date: item.SAMPLING_START,
-                sampling_end_date: item.SAMPLING_END,
-                risk_of_bias: item.OVERALL_RISK_OF_BIAS ? item.OVERALL_RISK_OF_BIAS[0] : null,
-                url: item.URL ? item.URL[0] : null,
-                include_in_n: item.INCLUDE_IN_N ? true : false,
-                estimate_grade: item.ESTIMATE_GRADE
+                include_in_n: true,
+                source_name: item.source_name,
+                lead_org: item.lead_org,
+                first_author: item.first_author,
+                source_type: item.source_type,
+                study_status: item.study_status,
+                study_type: item.study_type,
+                test_type: item.study_type,
+                specimen_type: Array.isArray(item.specimen_type) ? item.specimen_type : [item.specimen_type],
+                isotypes_reported: item.isotypes_reported,
+                sex: item.sex,
+                approving_regulator: item.approving_regulator,
+                country: item.country,
+                state: item.state,
+                city: item.city,
+                population_group: item.population_group,
+                age: item.age,
+                denominator: item.denominator_value,
+                seroprevalence: item.serum_pos_prevalence,
+                sample_size: null,
+                sampling_start_date: null,
+                sampling_end_date: item.sampling_end_date,
+                overall_risk_of_bias: item.overall_risk_of_bias,
+                estimate_grade: item.estimate_grade
             };
 
             return record;
         });
-
         // Remove timestamp from updated at string
-        const updated_at_words = response.updated_at!.split(" ", 4);
-        const updated_at_string = updated_at_words.join(" ");
 
-        return {
-            airtable_records,
-            updated_at: updated_at_string
-        };
+        return filtered_records;
     }
 
-    async getEstimateGrades(records: AirtableRecord[]) {
-        const formattedRecords = records.map(o => {
-            return {
-                country: o.country,
-                estimate_grade: o.estimate_grade,
-                serum_pos_prevalence: o.seroprevalence,
-                denominator: o.denominator
-            }
+    async getEstimateGrades(filters: Filters) {
+        const reqBodyFilters: Record<string, string[]> = {}
+        Object.keys(filters).forEach((o: string) => {
+            const filter = Array.from(filters[o as FilterType]);
+            reqBodyFilters[o] = filter as string[]
         })
-        const req_body = {
-            records: formattedRecords
+        delete reqBodyFilters['publish_date']
+        const date = filters['publish_date']
+        const endDate = date[1] ? formatISO(date[1] as Date) : formatISO(new Date());
+        const startDate = date[0] ? formatISO(date[0] as Date) : formatISO(add(new Date(endDate), { years: -2 }));
```

Comment:
use date parsing utility function here!

Reply:
On a second note, would be better for us to just be able to not provide a startDate to the backend if it's not specified. 

TODO: Validate if this functionality exists, if not then make ticket to do it
@atatmaja